packages/back-end/src/app.ts

```diff
@@ -563,7 +559,7 @@ app.delete(
   discussionsController.deleteComment
 );
 app.get("/discussions/recent/:num", discussionsController.getRecentDiscussions);
-app.post("/file/upload/:filetype", discussionsController.postImageUploadUrl);
+app.use("/upload", uploadsRouter);
```

Comment:
I haven't tested it, but I believe uploading when method is local is broken.

The staticFilesRouter, when  UPLOAD_METHOD is local, terminates the request to any `/upload` endpoint and stops it from reaching this point. 

Even if it's not broken, I won't be the only one to mistakenly think that, so it's worth making the interaction between the two `app.use("/upload")` calls more explicit and clear what's happening.